pkg/cli/prompt.go

```diff
@@ -28,29 +27,14 @@ func AskInput(r io.Reader, w io.Writer, prompt string) (string, error) {
 func AskPassword(f *os.File, w io.Writer, prompt string) (string, error) {
 	fd := int(f.Fd())
 
-	oldState, err := term.GetState(fd)
-	if err != nil {
-		return "", err
-	}
-	defer func() { _ = term.Restore(fd, oldState) }()
-
-	sigCh := make(chan os.Signal, 1)
-	signal.Notify(sigCh, os.Interrupt)
-	go func() {
-		for range sigCh {
-			_ = term.Restore(fd, oldState)
-			os.Exit(1)
-		}
-	}()
-
```

Comment:
We need to keep this because if you interrupt ReadPassword with e.g., `ctrl+c`, it will not echo anymore.
See: https://github.com/golang/go/issues/31180